docs/conf.py

```diff
@@ -19,6 +19,17 @@
 
 # Use the sunpy theme
 from sunpy_sphinx_theme.conf import *
+from sphinx_gallery.sorting import ExampleTitleSortKey
+
+import pyvista
+import numpy as np
+# necessary when building the sphinx gallery
+pyvista.BUILDING_GALLERY = True
+pyvista.OFF_SCREEN = True
+
+# Optional - set parameters like theme or window size
+pyvista.set_plot_theme('document')
+pyvista.rcParams['window_size'] = np.array([1024, 768]) * 2
```

Comment:
So is this still needed even if we can't get the gallery to display images on RTD?

Reply:
Sorry I forgot to get rid of this